package.json

```diff
@@ -3,10 +3,10 @@
   "displayName": "atlas",
   "description": "Connects vscode with different server adapters",
   "version": "0.0.2",
-  "publisher": "mohitsuman",
+  "publisher": "Red Hat Developers",
```

Comment:
The publisher name is incorrect here. It should be this: ``` "publisher": "redhat",```

Reply:
And also add and author field mentioning the following:
```"author": "Red Hat",```

Reply:
done